### PR TITLE
Add internal support for Realm 0.81.1, RecyclerRenderers 1.2.4, NanoTasks 1.1.0, MaterialishProgress 1.5 and MarkView 1.1.0

### PR DESCRIPTION
**Pull request by FabianTerhorst**

### CODE REVIEW

File: library/src/main/res/values/library_realm_strings.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <string name="library_Realm_author">Realm</string>
+    <string name="library_Realm_authorWebsite">https://github.com/realm</string>
+    <string name="library_Realm_libraryName">Realm</string>
+    <string name="library_Realm_libraryDescription">Realm is a mobile database: a replacement for SQLite &amp; ORMs</string>
+    <string name="library_Realm_libraryVersion">0.81.1</string>
+    <string name="library_Realm_libraryWebsite">https://realm.io/</string>
+    <string name="library_Realm_licenseId">apache_2_0</string>
+    <string name="library_Realm_isOpenSource">true</string>
+    <string name="library_Realm_repositoryLink">https://github.com/realm/realm-java</string>
+    <string name="library_Realm_classPath">io.realm</string>
```
Comment (mikepenz) [inline library/src/main/res/values/library_realm_strings.xml]: Should be a specific class. Best a class which will stay the same. like: `io.realm.Realm`

Comment (FabianTerhorst) [inline library/src/main/res/values/library_realm_strings.xml]: changed realm class path to io.realm.Realm

**Comment by mikepenz** [comment]
@FabianTerhorst now you added 2 more libs. please modify the classpath for them too. 

thanks :)

**Comment by mikepenz** [comment]
Thank you so much

**Pull request merged**